hathor/indexes/tips_index.py

```diff
@@ -61,3 +65,27 @@ def update_tx(self, tx: BaseTransaction, *, relax_assert: bool = False) -> None:
     @abstractmethod
     def __getitem__(self, index: float) -> Set[Interval]:
         raise NotImplementedError
+
+    def iter_mempool_tips_like(self, tx_storage: 'TransactionStorage') -> Iterator[Transaction]:
```

Comment:
Missing docstring.

Reply:
Added docstring to the method, but it was also moved.

Reply:
Thanks!